test/tests/CookieBanner-test.js

```diff
@@ -108,7 +108,32 @@ describe('CookieBanner', () => {
     expect(banner.length).toBe(0, 'cookie banner is being displayed');
 
     const _myComponent = TestUtils.scryRenderedDOMComponentsWithClass(cookieWrapper, 'my-component');
-    expect(_myComponent.length).toBe(1, 'cookie banner is not displaing custom child component');
+    expect(_myComponent.length).toBe(1, 'cookie banner is not displaying custom child component');
+  });
+
+  it('should be replaced with custom child component using function', () => {
+
+    const MyOtherComponent = React.createClass({
```

Comment:
Actually, sorry, I just realized this is not your fault but you just copy-pasted the example above. This is our mistake, and I'll fix it in a subsequent PR.!